commands/utils/dom.js

```diff
@@ -88,24 +88,79 @@ async function sendDoM(storybookUrl, stories, storybookConfig, options) {
                 console.log('[smartui] Build failed: Error: ', error.response.data.message);
             } else {
                 console.log('[smartui] Build failed: Error: ', error.message);
-            }       
+            }
         });
-    
-    fs.rm('doms', {recursive: true}, (err) => {
+
+    fs.rm('doms', { recursive: true }, (err) => {
         if (err) {
             return console.error(err);
         }
     });
 };
 
+async function fetchDOM(screenshots, options) {
+    console.log("fetchDOM started")
+    const createBrowser = require('browserless')
+    const browser = createBrowser()
+
+    if (!fs.existsSync('doms')) {
+        fs.mkdir('doms', (err) => {
+            if (err) {
+                console.error(err);
+                process.exit(constants.ERROR_CATCHALL);
+            }
+        });
+    }
+
+    //TODO: Make this async
+    for (const screenshot of screenshots) {
+        console.log(screenshot)
+        let id = generateId(screenshot.name)
+        console.log(id)
+        screenshot.id = id
+        const browserless = await browser.createContext()
+
+        const html = await browserless.html(screenshot.url)
```

Comment:
Do we need a check that screenshot.url is valid/reachable?

Reply:
right now, I'm not checking that, but will add that with other improvements